x-pack/plugins/transform/public/app/sections/transform_management/transform_management_section.tsx

```diff
@@ -77,6 +79,23 @@ export const TransformManagement: FC = () => {
   // Call useRefreshInterval() after the subscription above is set up.
   useRefreshInterval(setBlockRefresh);
 
+  const unauthorizedTransformsWarning = useMemo(() => {
+    const unauthorizedCnt = transforms.filter((t) => needsReauthorization(t)).length;
+    return unauthorizedCnt > 0 ? (
+      <>
+        <EuiCallOut
+          color="warning"
+          title={i18n.translate('xpack.transform.transformList.unauthorizedTransformsCallout', {
+            defaultMessage:
+              '{unauthorizedCnt, plural, one {A transform was installed but requires} other {# transforms were installed but require}} additional permissions to run. You must have the transform_admin built-in role or manage_transform cluster privileges to reauthorize and start operations.',
```

Comment:
IMO this can be shortened:
```suggestion
              '{unauthorizedCnt, plural, one {A transform was installed but requires more permissions to run. You must have manage_transform cluster privileges to start it.} other {# transforms were installed but require more permissions to run. You must have manage_transform cluster privileges to start them.}} ',
```

Reply:
Updated here [`3f64b1f` (#154736)](https://github.com/elastic/kibana/pull/154736/commits/3f64b1f53667da00f811cc2579eb31decf81fafb)